openshift/roles/client/tasks/main.yml

```diff
@@ -22,7 +22,7 @@
     - heketi-templates
 
 - name: get openshift-ansible release 1.3
-  git: repo=https://github.com/openshift/openshift-ansible dest=/home/vagrant/openshift-ansible version=3cc5b0562de1b8a2334104542c15a6cce67b68bf
+  git: repo=https://github.com/openshift/openshift-ansible dest=/home/vagrant/openshift-ansible version=release-1.3
```

Comment:
Not 100% sure about this: release-1.3 is the 1.3 release branch. So this will be a moving target. Wouldn't it be better to take a specific release, but not with a githash, but maybe wit a tag, like `openshift-ansible-3.3.68-1` which is the latest release tag in the release-1.3 branch.